src/Umbraco.Cms.Api.Management/Factories/IPartialViewSnippetPresentationFactory.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Umbraco.Cms.Api.Management.ViewModels.PartialView.Snippets;
+using Umbraco.Cms.Core.Snippets;
+
+namespace Umbraco.Cms.Api.Management.Factories;
+
+public interface IPartialViewSnippetPresentationFactory
+{
+    PartialViewSnippetItemResponseModel CreateSnippetItemResponseModel(string fileName);
+
+    PartialViewSnippetResponseModel CreateSnippetResponseModel(PartialViewSnippet snippet);
```

Comment:
I wonder if this should be moved downwards to when we seed the Snippet collection. That way the Snippet could have a Name or DisplayName and an identifier (currently the name).

Reply:
Yep, you're absolutely right. I have now rewritten snippets more or less entirely to this end.